pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -407,3 +418,13 @@ func getSystemReservedMemory(machineInfo *cadvisorapi.MachineInfo, nodeAllocatab
 
 	return reservedMemoryConverted, nil
 }
+
+// GetAllocatableMemory returns the amount of allocatable memory for each NUMA node
+func (m *manager) GetAllocatableMemory() []state.Block {
+	return m.allocatableMemory
```

Comment:
Is it OK for this to be cached like this? I guess the assumption is that _if_ the amount of allocatable memory was ever changed, then the kubelet would be restarted and this value recalculated.

Reply:
Exactly, to change the allocatable you should update the `reservedMemory` parameter and it will require the kubelet restart.